docs/template_instructions.rst

```diff
@@ -1,101 +1,59 @@
  | # # # Distribution Statement A. Approved for public release. Distribution unlimited.
- | # # # 
+ | # # #
  | # # # Author:
  | # # # Naval Research Laboratory, Marine Meteorology Division
- | # # # 
+ | # # #
  | # # # This program is free software: you can redistribute it and/or modify it under
  | # # # the terms of the NRLMMD License included with this program. This program is
  | # # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
  | # # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
  | # # # for more details. If you did not receive the license, for more information see:
  | # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
 
+###############################################################
+Instructions for setting up test data repository from templates
+###############################################################
+
+#. Replace all instances of @mydatatype@ in README.md with your actual data type name
+    (as in test_data_@mydatatype@)
+#. Identify any additional instances of @ within README.md, and replace with appropriate content.
+#. Update VERSION with desired version number
+  * *vim $GEOIPS_TESTDATA_DIR/test_data_@mydatatype@/VERSION*
+  * No comments of other contents allowed within VERSION file
+  * GeoIPS as a whole follows PEP-440 semantic versioning: https://peps.python.org/pep-0440/
+#. Create "data" subdirectory, and populate with appropriate datasets
+  * git lfs track \*.@ext@  # Ensure all large data files are tracked with git lfs
+  * If you compress any datasets, ensure the "uncompress\_test\_data.sh" script will uncompress them.
+#. Create "bg\_data" subdirectory, and populate with appropriate datasets
+  * git lfs track \*.@ext@  # Ensure all large data files are tracked with git lfs
+  * If you compress any datasets, ensure the "uncompress\_test\_data.sh" script will uncompress them.
+#. Update CHANGELOG.md with updates / description of datasets
+  * *vim $GEOIPS_TESTDATA_DIR/test_data_@mydatatype@/CHANGELOG.md*
```

Comment:
I think this would be better written as:
```suggestion
#. Replace all instances of @mydatatype@ in README.md with your actual data type name
   (as in test_data_@mydatatype@)

#. Identify any additional instances of @ within README.md, and replace with appropriate content.

#. Update VERSION with desired version 

   - ``vim $GEOIPS_TESTDATA_DIR/test_data_@mydatatype@/VERSION``
   - *No comments of other contents allowed within VERSION file*
   - *GeoIPS as a whole follows PEP-440 semantic versioning: https://peps.python.org/pep-0440/*
   
#. Create "data" subdirectory, and populate with appropriate datasets

   - ``git lfs track \*.@ext@  # Ensure all large data files are tracked with git lfs``
   - *If you compress any datasets, ensure the "uncompress\_test\_data.sh" script will uncompress them.*
   
#. Create "bg\_data" subdirectory, and populate with appropriate datasets

   - ``git lfs track \*.@ext@  # Ensure all large data files are tracked with git lfs``
   - *If you compress any datasets, ensure the "uncompress\_test\_data.sh" script will uncompress them.*
   
#. Update CHANGELOG.md with updates / description of datasets

   - ``vim $GEOIPS_TESTDATA_DIR/test_data_@mydatatype@/CHANGELOG.md``
```